src/components/charts/util/ChartValueSelect.tsx

```diff
@@ -12,6 +12,7 @@ export interface ChartValuesSelectProps {
     chartValuesList: ChartValuesType[];
     hideVersionFromLabel?: boolean;
     className?: string;
+    hideCreateNewOption?: boolean
```

Comment:
Instead of this can we rename it to `showCreateNewOption`

Reply:
In most cases we will be showing this createNew option that 's why created hide variable